plans/rerun.fmf

```diff
@@ -719,6 +719,7 @@
             automatically selects the subscription.
         discover+:
             filter: tag:auto-attach-pool
+<<<<<<< HEAD
```

Comment:
Please resolve this git conflict resolution leftover.

Reply:
Thanks I'll remove this